lib/elixir_search_extractor/search_keyword/queries/keyword_query.ex

```diff
@@ -0,0 +1,13 @@
+defmodule ElixirSearchExtractor.SearchKeyword.Queries.KeywordQuery do
+  import Ecto.Query
+  alias ElixirSearchExtractor.FileUpload.Schemas.KeywordFile
+  alias ElixirSearchExtractor.SearchKeyword.Schemas.Keyword
+
+  def user_keywords(user) do
+    from keyword in Keyword,
+      join: file in KeywordFile,
+      on: keyword.keyword_file_id == file.id,
+      where: file.user_id == ^user.id,
+      order_by: [desc: :inserted_at]
```

Comment:
For query, we prefer the pipe-based syntax if possible. The above query could be rewrite to:

```elixir
Keyword
|> join(:inner, [k], kf in assoc(k, :keyword_file)
|> where([_, kf], kf.user_id == ^user.id)
|> order_by([k, _], desc: k. inserted_at)
```

https://nimblehq.co/compass/development/code-conventions/elixir-phoenix/#database